examples/llm/mcloud/mcli-1b-custom.yaml

```diff
@@ -1,11 +1,18 @@
 integrations:
+# To point to MosaicML's public repo:
 - integration_type: git_repo
   git_repo: mosaicml/examples
-  git_branch: v0.0.3 # use your branch
-  # git_commit: # OR use your commit hash
```

Comment:
@abhi-mosaic No it should be there! Just moved it down into the custom repo section

Reply:
oh ok I see the comment now. I think it might still be a bit more clear to give it a newline, I think people have a tendency to just skim and select-delete everything after `git_branch:` and then if they try to put a commit hash it would fail.

Reply:
Basically like "probability of customers pinging us in Slack" goes down :)